test/stale.test.js

```diff
@@ -195,4 +195,30 @@ describe('stale', () => {
       expect(stale.getClosable).not.toHaveBeenCalled()
     }
   )
+
+  test(
+    'should count weekends for age by default',
+    async () => {
+      let stale = new Stale(github, {perform: true, owner: 'probot', repo: 'stale', logger: robot.log})
+      stale.config.skipWeekends = false
+      const now = Date.now()
+      Date.now = jest.genMockFunction().mockReturnValue(now)
+      const sinceDate = stale.since(7)
+      expect(sinceDate).toEqual(new Date(now - 7 * 24 * 60 * 60 * 1000))
```

Comment:
Could you change this to use a fixed date like the next test?

Reply:
+1